packages/ui/src/2_molecules/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+
+import { RadioButton } from '../RadioButton';
+import styles from './RadioButtonGroup.module.css';
+import {
+  RadioButtonOption,
+  RadioButtonGroupProps,
+} from './RadioButtonGroup.types';
+
+export const RadioButtonGroup: FC<RadioButtonGroupProps> = ({
+  label,
+  options,
+  onChange,
+  dataLayoutId,
+}) => (
+  <fieldset className={styles.radioButtonGroup} data-layout-id={dataLayoutId}>
+    <legend>{label}</legend>
+    {options.map(
+      ({ label, name, disabled, labelInfo }: RadioButtonOption, index) => {
+        const id = `radio-option-${label}`;
+        return (
+          <RadioButton
+            id={id}
+            key={id}
+            label={label}
+            value={label}
+            name={name}
+            disabled={disabled}
+            defaultChecked={index === 0}
```

Comment:
How about getting it as prop for RadioButtonGroupProps? (and if it was undefined none of the options be pre selected)